org.deidentifier.arx.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: org.deidentifier.arx
+runtime: org.gnome.Platform
```

Comment:
do you really need the gnome platform given this is Java

Reply:
yes. It doesn't run otherwise. I've tested it and complained on their bug tracker: https://github.com/arx-deidentifier/arx/issues/416
It's a typical Java stacktrace: super long but no information as to what's going on.

Reply:
The filename contains Gtk. So I guess it uses things from Gtk at one point.

Reply:
Gtk3 is in the freedestkop runtime. Has always been.

Reply:
> yes. It doesn't run otherwise. I've tested it and complained on their bug tracker: [arx-deidentifier/arx#416](https://github.com/arx-deidentifier/arx/issues/416) It's a typical Java stacktrace: super long but no information as to what's going on.

yuk.

ok